groups.go

```diff
@@ -71,6 +71,7 @@ type Group struct {
 	LDAPCN                         string           `json:"ldap_cn"`
 	LDAPAccess                     AccessLevelValue `json:"ldap_access"`
 	LDAPGroupLinks                 []*LDAPGroupLink `json:"ldap_group_links"`
+	FileTemplateProjectId          int              `json:"file_template_project_id"`
```

Comment:
Please place this between `FullPath` and `ParentID` following the ordering in the docs. And for the naming:
```suggestion
	FileTemplateProjectID          int              `json:"file_template_project_id"`
```

Reply:
Thanks @svanharmelen ! I fixed this issue along with updating the test as well, feel free to re-review 😄